[2019-04-05T10:24:24,938][FATAL][logstash.runner          ] An unexpected error occurred! {:error=>#<SystemCallError: Unknown error (SystemCallError) - <STDOUT>>, :backtrace=>["org/jruby/RubyIO.java:1457:in `write'", "org/jruby/RubyIO.java:1428:in `write'", "/Users/runze/Desktop/Yuqi/2019_spring/Software_modeling_development_UML/Assignment3/Assignment3_Yuqi/logstash-6.2.2/vendor/bundle/jruby/2.3.0/gems/logstash-output-stdout-3.1.3/lib/logstash/outputs/stdout.rb:44:in `block in multi_receive_encoded'", "org/jruby/RubyArray.java:1734:in `each'", "/Users/runze/Desktop/Yuqi/2019_spring/Software_modeling_development_UML/Assignment3/Assignment3_Yuqi/logstash-6.2.2/vendor/bundle/jruby/2.3.0/gems/logstash-output-stdout-3.1.3/lib/logstash/outputs/stdout.rb:43:in `multi_receive_encoded'", "/Users/runze/Desktop/Yuqi/2019_spring/Software_modeling_development_UML/Assignment3/Assignment3_Yuqi/logstash-6.2.2/logstash-core/lib/logstash/outputs/base.rb:90:in `multi_receive'", "/Users/runze/Desktop/Yuqi/2019_spring/Software_modeling_development_UML/Assignment3/Assignment3_Yuqi/logstash-6.2.2/logstash-core/lib/logstash/output_delegator_strategies/single.rb:15:in `block in multi_receive'", "org/jruby/ext/thread/Mutex.java:148:in `synchronize'", "/Users/runze/Desktop/Yuqi/2019_spring/Software_modeling_development_UML/Assignment3/Assignment3_Yuqi/logstash-6.2.2/logstash-core/lib/logstash/output_delegator_strategies/single.rb:14:in `multi_receive'", "/Users/runze/Desktop/Yuqi/2019_spring/Software_modeling_development_UML/Assignment3/Assignment3_Yuqi/logstash-6.2.2/logstash-core/lib/logstash/output_delegator.rb:49:in `multi_receive'", "/Users/runze/Desktop/Yuqi/2019_spring/Software_modeling_development_UML/Assignment3/Assignment3_Yuqi/logstash-6.2.2/logstash-core/lib/logstash/pipeline.rb:479:in `block in output_batch'", "org/jruby/RubyHash.java:1343:in `each'", "/Users/runze/Desktop/Yuqi/2019_spring/Software_modeling_development_UML/Assignment3/Assignment3_Yuqi/logstash-6.2.2/logstash-core/lib/logstash/pipeline.rb:478:in `output_batch'", "/Users/runze/Desktop/Yuqi/2019_spring/Software_modeling_development_UML/Assignment3/Assignment3_Yuqi/logstash-6.2.2/logstash-core/lib/logstash/pipeline.rb:430:in `worker_loop'", "/Users/runze/Desktop/Yuqi/2019_spring/Software_modeling_development_UML/Assignment3/Assignment3_Yuqi/logstash-6.2.2/logstash-core/lib/logstash/pipeline.rb:385:in `block in start_workers'"]}
[2019-04-05T10:24:25,486][ERROR][org.logstash.Logstash    ] java.lang.IllegalStateException: org.jruby.exceptions.RaiseException: (SystemCallError) Unknown error (SystemCallError) - <STDOUT>
